tests/test-volumes-absent-helper.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Make sure all volumes in pool.volumes are absent
+  assert:
+    that:
+      - current_volume.mount_point not in fstab_buffer.stdout
+      - "{{ ansible_mounts|selectattr('fstype', 'equalto', current_volume.fs_type)|list|length==1 if current_volume.fs_type is defined else
+            ansible_mounts|selectattr('fstype', 'equalto', 'xfs')|list|length==1 }}"
+    msg: "The pool volume {{ current_volume.name }} was incorrectly managed."
+  loop: "{{ current_pool.volumes }}"
+  loop_control:
+    loop_var: current_volume
```

Comment:
I was having difficulties using multiple statements in an `if` comparison as it would need to be `if (current_volume.state is defined and current_volume in 'present') else...`, and I was unable to get this to run successfully.

Reply:
Could either find a more elegant solution or just hardcode the `set_fact` for volume.state to be `absent` when `volume.state is undefined` in the main task flow

Reply:
I was mistaken, I was able to get it to work out.